spi/src/main/java/org/arquillian/cube/spi/metadata/HasMappedPorts.java

```diff
@@ -0,0 +1,13 @@
+package org.arquillian.cube.spi.metadata;
+
+
+public interface HasMappedPorts extends CubeMetadata {
+    
+    String getIP();
+
+    Integer forTargetPort(int targetPort);
+
+    void createProxies() throws Exception;
```

Comment:
I would have moved create/destroy to the impl to avoid exposing it in SPI. The control of the creation and destruction of the Mapped Ports should be transparent and can be handled internally within the Cube.